libcst/codemod/tests/test_codemod_cli.py

```diff
@@ -38,3 +39,34 @@ def test_codemod_formatter_error_input(self) -> None:
                 "error: cannot format -: Cannot parse: 13:10:     async with AsyncExitStack() as stack:",
                 rlt.stderr.decode("utf-8"),
             )
+
+    def test_warning_messages_several_files(self) -> None:
+        mod = """
+        def baz() -> str:
+            return "{}: {}".format(*baz)
+
+        def foobar() -> str:
+            return "{x}: {y}".format(**baz)
+        """
+        with tempfile.TemporaryDirectory() as root:
+            p = Path(root)
+            (p / "mod1.py").write_text(CodemodTest.make_fixture_data(mod))
+            (p / "mod2.py").write_text(CodemodTest.make_fixture_data(mod))
+            (p / "mod3.py").write_text(CodemodTest.make_fixture_data(mod))
+            rlt = subprocess.run(
+                [
+                    "python",
+                    "-m",
+                    "libcst.tool",
+                    "codemod",
+                    "convert_format_to_fstring.ConvertFormatStringCommand",
+                    p,
+                ],
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+            )
+            # Each module will generate 2 warnings, so we should get 6 warnings in total
+            self.assertIn(
+                "- 6 warnings were generated.",
```

Comment:
I take it 6 because there are 2 per file - would it have been 2 + 4 + 6 = 12 prior to this fix?